src/mono/netcore/sample/wasm/browser/runtime.js

```diff
@@ -0,0 +1,40 @@
+
+var Module = { 
+    onRuntimeInitialized: function () {
+        config.loaded_cb = function () {
+            App.init ();
+        };
+        config.fetch_file_cb = function (asset) {
+            if (typeof window != 'undefined') {
```

Comment:
I don't think this will ever not be true in a browser context so we don't need the `else`.